SparkPost/2web2ui
# FE-1250 - Add Recipient Lists Empty State

### What Changed

- Added new empty state for Recipient Lists
- Note: Copy does not match mocks, it has been updated

### How To Test

- Run the app
- Enable account ui flag: "allow_empty_states"
- With no recipient lists and hibana enabled, verify you see the empty state
- With recipient lists make sure you see the muted banner if it has not previously been dismissed

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
